src/App/Program.fs

```diff
@@ -66,9 +66,12 @@ type MissionOptions
         apiRateLimit: int,
         pubnetData: string option,
         tier1Keys: string option,
+        opCountDistribution: string option,
         installNetworkDelay: bool option,
         simulateApplyDuration: seq<string>,
         simulateApplyWeight: seq<string>,
+        loadGenOpCount: seq<string>,
```

Comment:
Do we need `loadGenOpCount` and `loadGenOpCountDistribution`? That seems to be the same thing as `opCountDistribution`, just different format.